wire-ios/Configuration/url.json

```diff
@@ -1,5 +1,23 @@
 {
-    "wireAppOnItunes": "https://geo.itunes.apple.com/us/app/wire/id930944768?mt=8",
+    "appOnItunes": "https://geo.itunes.apple.com/us/app/wire/id930944768?mt=8",
     "support": "https://support.wire.com",
-    "randomProfilePictureSource": "https://source.unsplash.com/800x800/?landscape"
+    "searchSupport": "https://wire.com/support/username",
```

Comment:
Is this link correct? Shouldn't it be `https://support.wire.com/hc`?

Reply:
I took this link from an old implementation and the result is that it redirects to `https://support.wire.com/hc`.
I updated the link.